modules/shortcodes/codepen.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/*
+ * CodePen embed
+ *
+ * example URL: http://codepen.io/css-tricks/pen/wFeaG
+*/
+
+// Register oEmbed provider
+wp_oembed_add_provider( '#http[s]{0,1}://codepen.io/([^/]+)/pen/([^/]+)[/]{0,1}#', 'https://codepen.io/api/oembed', true );
```

Comment:
@aduth Thanks! I implemented your changes in a7e12d258a2275cc759ee180a43ee24908682b94